tests/test_ext_viewcode.py

```diff
@@ -67,10 +67,10 @@ def find_source(app, modname):
         if modname == 'not_a_package':
             source = (app.srcdir / 'not_a_package/__init__.py').text()
             tags = {
-                'func1': ('def', 3, 3),
-                'Class1': ('class', 3, 3),
-                'not_a_package.submodule.func1': ('def', 3, 3),
-                'not_a_package.submodule.Class1': ('class', 3, 3),
+                'func1': ('def', 1, 1),
+                'Class1': ('class', 1, 1),
+                'not_a_package.submodule.func1': ('def', 1, 1),
+                'not_a_package.submodule.Class1': ('class', 1, 1),
```

Comment:
Line numbers are hard coded in test case. So I just updated them all.

Reply:
Oh damn. So simple. Thanks!